sample-cap/srv/foundation-models/azure-openai-service.ts

```diff
@@ -1,8 +1,7 @@
-import { Request } from '@sap/cds';
 import { AzureOpenAiChatClient } from '@sap-ai-sdk/foundation-models';
 
 export default class AzureOpenAiService {
-  async chatCompletion(req: Request) {
```

Comment:
Fyi, as I was constantly facing (flaky) type issues with CAP, saying `Request` cannot be resolved even when I install additionally `@cap-js/cds-types`  as dev dependency, I turned it into `any`. It is out-of-scope of this PR.